projects/storefrontapp/src/environments/environment.ts

```diff
@@ -21,7 +21,7 @@ import { Environment } from './models/environment.model';
 
 export const environment: Environment = {
   production: false,
-  occBaseUrl: buildProcess.env.CX_BASE_URL,
+  occBaseUrl: 'https://spartacus-dev0.eastus.cloudapp.azure.com:9002',
```

Comment:
don't forget to revert it back before merging.

Reply:
Just checking, was temp config thing reverted?

Reply:
reverted